fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -448,6 +453,7 @@ public String submitImport(String inputFormat, String inputSource, List<Input> i
         builder.applicationName(properties.get(BulkDataConfigUtil.APPLICATION_NAME));
         builder.moduleName(properties.get(BulkDataConfigUtil.MODULE_NAME));
         builder.cosBucketName(properties.get(BulkDataConfigUtil.JOB_PARAMETERS_BUCKET));
+        builder.cosBucketNameOo(properties.get(BulkDataConfigUtil.JOB_PARAMETERS_BUCKET));
```

Comment:
better use OperationOutcome instead of Oo?

Reply:
Fixed in next commit

Reply:
Fixed in next commit